.checkstyle/suppressions.xml

```diff
@@ -8,4 +8,7 @@
 
     <!-- Note that [/\\] must be used as the path separator for cross-platform support -->
 
+    <suppress checks="NPathComplexity"
+              files="io[/\\]strimzi[/\\]kafka[/\\]bridge[/\\]Application.java"/>
+
```

Comment:
why did you need that?

Reply:
I added there a check whether the config file is specified and that increased `NPathComplexity`.